src/main/webapp/app/components/panels/CompanionDiagnosticDevicePanel.tsx

```diff
@@ -86,16 +99,17 @@ const CompanionDiagnosticDevicePanel: React.FunctionComponent<StoreProps> = prop
           <SidebarMenuItem>Add Biomarker Association</SidebarMenuItem>
           <SidebarMenuItem>
             <GeneSelect
+              isMulti={false}
```

Comment:
This is new to me. Why are we making the prop required when it's not in react-select?

Reply:
It's to help the type checker know that we are using single values instead of arrays. You could get the same effect by defaulting a value for isMulti in the GeneSelect component itself.

Reply:
@jfkonecn the react-select does not make this required field, we also should not. Could you elaborate why adding this would help with type checker?

Reply:
@zhx828 basically removing `isMulti` makes typescript confused about what the underlying value. It thinks it can be either a single value or an array if you leave `isMulti` undefined which is clearly wrong. I'll see if I can make it work without the property.